.travis.yml

```diff
@@ -53,6 +53,21 @@ matrix:
       - BUILD_TYPE=Release
       - CXXFLAGS="-std=c++2a"
       - CMAKE_VERSION=3.7.2
+  - os: linux
+    compiler: 'g++-9'
+    dist: bionic
+    addons:
+      apt:
+        sources:
+          - sourceline: ppa:ubuntu-toolchain-r/test
+        packages: ['g++-9', 'cmake']
```

Comment:
I couldn't use the `<< : *linux-gcc-9` trick as I needed to customize the package list